src/test/resources/release-notes-file.md

```diff
@@ -0,0 +1,11 @@
+### New features
+
+* [JIRA-TICKET](jira-link): Some feature
```

Comment:
No regex. On GitHub release I take RELEASE-NOTES.md file and put its content as release notes to the new release. Then override RELEASE-NOTES.md with standard template. Then you may start populating the new file for future release.